.gitignore

```diff
@@ -0,0 +1,51 @@
+# Compiled source #
+###################
+*.com
+*.class
+*.dll
+*.exe
+*.o
+*.so
+*.pyc
+
+# Packages #
+############
+# it's better to unpack these files and commit the raw source
+# git has its own built in compression methods
+*.7z
+*.dmg
+*.gz
+*.iso
+*.jar
+*.rar
+*.tar
+*.zip
+
+# Logs and databases #
+######################
+*.log
+*.sql
+*.sqlite
+
+# LaTeX Generated Files #
```

Comment:
It would be much better for this to only exclude temporary files generated by LaTeX.  (For example, someone with a `sql` or `sqlite` file used to store experiment results and automatically generate figures might be surprised to find it excluded from version control.)